index.js

```diff
@@ -5,7 +5,8 @@ module.exports = fp(
   async function (fastify) {
     await fastify.register(require('@fastify/static'), {
       root: path.join(__dirname, 'dist'),
-      prefix: '/fastify-overview-ui/'
+      prefix: '/fastify-overview-ui/',
+      decorateReply: false
```

Comment:
The problem with this is that this is needed https://github.com/nearform/fastify-overview-ui/pull/16/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R10-R12. This change fixes an issue. For some reason @fastify/static doesn't seem to be serving the index.html file

Reply:
I need to check better that PR.
I'm using the v1 right now and, apart this option, works like a charm

Reply:
oh that's weird. so here's what happened. when I released v1.0.0 I tested it and it seemed to work. when I picked it up again I couldn't get it to work instead and came to the conclusion that I did a mistake and that index.html wasn't being properly served by @fastify/static. to this moment to me it still doesn't work without the changes in #16

Reply:
Let me share the project I used during a recent talk:

[firenze-commit.zip](https://github.com/nearform/fastify-overview-ui/files/8933759/firenze-commit.zip)